view/lang/fr/messages.po

```diff
@@ -9421,8 +9421,8 @@ msgid "Your invitation code: "
 msgstr "Votre code d'invitation :"
 
 #: src/Module/Register.php:152
-msgid "Your Display Name (as you would like it to be displayed on this system"
-msgstr "Votre Nom d'Affichage (tel que vous souhaiteriez l'afficher sur ce système"
+msgid "Your Display Name (as you would like it to be displayed on this system)"
+msgstr "Votre Nom d'Affichage (tel que vous souhaiteriez l'afficher sur ce système)"
```

Comment:
For reference, this translation was also missing the closing parenthesis. I added it here, but happy to revert if such changes should be made in Transifex instead.

Reply:
Resolving this conversation as I am no longer going to edit the files in `view/lang/` directly, as recommended in code review.